src/test/java/org/jenkinsci/plugins/workflow/graph/FlowNodeTest.java

```diff
@@ -54,21 +55,20 @@
 import org.junit.Test;
 import org.kohsuke.stapler.DataBoundConstructor;
 
+import static org.hamcrest.Matchers.hasItem;
 import static org.hamcrest.Matchers.hasSize;
 import static org.junit.Assert.*;
 
 import org.junit.Rule;
 import org.junit.runners.model.Statement;
 import org.jvnet.hudson.test.Issue;
 import org.jvnet.hudson.test.LoggerRule;
-import org.jvnet.hudson.test.JenkinsRule;
 import org.jvnet.hudson.test.RestartableJenkinsRule;
 import org.jvnet.hudson.test.TestExtension;
 
 public class FlowNodeTest {
 
     @Rule public RestartableJenkinsRule rr = new RestartableJenkinsRule();
-    @Rule public JenkinsRule r = new JenkinsRule();
```

Comment:
I was getting some curious error messages at the end of tests and then noticed this 😄